trestle/tasks/xlsx_to_oscal_component_definition.py

```diff
@@ -692,10 +694,11 @@ def _get_parameter_name_and_description(self, work_sheet: t_work_sheet, row: t_r
                 parameter_parts = combined_values.split(' ', 1)
             else:
                 parameter_parts = combined_values
-            if len(parameter_parts) != 2:
-                raise RuntimeError(f'row {row} col {col} unable to parse')
-            name = parameter_parts[1].strip()
-            description = parameter_parts[0].strip()
+            if len(parameter_parts) == 2:
+                name = parameter_parts[1].strip()
+                description = parameter_parts[0].strip()
+            else:
+                logger.info(f'row {row} col {col} invalid value')
         value = name, description
```

Comment:
should this be warning?

Reply:
No, this in in keeping with the other like messages that are given as informational. I suppose we could consider changing them all to warning. I think it is fine for now.